tests/queries/0_stateless/03463_numeric_indexed_vector_overflow.sql

```diff
@@ -54,3 +54,8 @@ SELECT groupNumericIndexedVector(x, y) FROM values('x Nullable(Int32), y Nullabl
 SELECT groupNumericIndexedVector(x, y) FROM values('x Nullable(Int32), y Nullable(Float64)', (1, 0), (3, Null), (3, 2), (0, 0), (5, 1));
 SELECT groupNumericIndexedVector(x, y) FROM values('x Nullable(Int32), y Nullable(Float64)', (1, 0), (3, inf), (3, 2), (0, 0), (5, 1)); -- { serverError INCORRECT_DATA }
 SELECT groupNumericIndexedVector(x, y) FROM values('x Nullable(Int32), y Nullable(Float64)', (1, 0), (3, -inf), (3, 2), (0, 0), (5, 1)); -- { serverError INCORRECT_DATA }
+
+
```

Comment:
(You probably want to add a small code comment what this block tests + a ref to the GitHub issue)